src/NewCommand.php

```diff
@@ -133,23 +136,11 @@ protected function makeFilename()
      * Download the temporary Zip to the given file.
      *
      * @param  string  $zipFile
-     * @param  string  $version
+     * @param  string  $filename
      * @return $this
      */
```

Comment:
The version is now passed to the `getFilename` method to properly get the required filename